src/features/post/inFeed/compact/CompactPost.tsx

```diff
@@ -34,6 +38,13 @@ const Aside = styled.div`
   font-size: 0.8em;
 `;
 
+export const AnnouncementIcon = styled(IonIcon)`
```

Comment:
I believe this is duplicated in large and compact posts. Can you import it from LargePost, or extract it to src/psot/inFeed/shared.ts (or something)?

Reply:
Updated so that `CompactPost.tsx` imports `AnnouncementIcon` the same way `LargePost.tsx` does.

I pushed the commit up to my repo. My experience is with gitlab, not github but pretty sure that's all I need to do to prep this for more review, but if I'm missing a step, let me know.

Reply:
Just realized I left an unnecessary import in the file; removed that.